docs/src/pages/configurations/default-config/index.md

```diff
@@ -56,6 +56,11 @@ import './styles.css';
 
 > **Note:** this is plain CSS only. If you need a preprocessor like SASS, you need to [customize the webpack config](/configurations/custom-webpack-config/).
 
+> **Warning:** storybooks for projects that use Angular CLI must use the inline loader syntax:
+> ```js
+> import '!style-loader!css-loader!./styles.css';
```

Comment:
I would say that this is also available by adding an extra rule to the extended webpack config.

Reply:
Do you mean like [you talked about here](https://github.com/storybooks/storybook/issues/2647#issuecomment-355598718)? Is this really something we want to explicitly recommend? It relies on manipulating rules that are part of the private Angular CLI implementation, and could break if Angular changes something.

Reply:
I think bringing here a concrete example is not needed. Just to mention about a feature storybook has (extended webpack config). I think that people just need to know about all the possibilities they have.